flutter/test/integrations/init_native_sdk_integration_test.dart

```diff
@@ -54,8 +54,12 @@ void main() {
         'sendDefaultPii': false,
         'enableOutOfMemoryTracking': true,
         'enableNdkScopeSync': false,
-        'enableAutoPerformanceTracking': true,
-        'sendClientReports': true
+        'enableAutoPerformanceTracing': true,
+        'sendClientReports': true,
+        'sdk': {
+          'name': 'sentry.dart',
```

Comment:
I believe here should be `sentry.dart.flutter`?

Reply:
It is only set in the real sdk code, not in the test. I updated the test classes to better reflect the flutter sdk.